bootstrap/vagrant/config-default.sh

```diff
@@ -1 +1,2 @@
 #!/bin/bash
+echo ""
```

Comment:
commited by accident, shellchecker was complaining about empty file, probably want to remove that file and check if exits before source it